packages/devtools/src/components/Logger/Logger.ts

```diff
@@ -0,0 +1,5 @@
+import { ConsoleLogger } from '@microsoft/spark.common/logging';
```

Comment:
should this be here if its not a react component?

Reply:
Yep! This is just instantiating the logger with the name so any React component can import it and use levels beyond `'@spark/devtools'`